invertedai/cosimulation.py

```diff
@@ -111,7 +111,8 @@ def ego_agent_mask(self, value):
     @property
     def ego_states(self):
         """
-        The predicted states for ego_agents only.
+        Returns the current states of ego agents in order.
```

Comment:
Why did you change "predicted" to "current"? I used "predicted" to emphasize that this is one step ahead of the local simulator.